common-queries/src/main/scala/queries/schemas/odb/ObsQueries.scala

```diff
@@ -126,45 +105,66 @@ object ObsQueries {
     val constraintGroups = Focus[ObsSummariesWithConstraints](_.constraintGroups)
   }
 
-  private val queryToObsSummariesWithConstraintsGetter
-    : Getter[ProgramObservationsQuery.Data, ObsSummariesWithConstraints] = data =>
-    ObsSummariesWithConstraints(
-      KeyedIndexedList.fromList(
-        data.observations.matches.map(mtch =>
-          ObsSummaryWithTitleConstraintsAndConf(
-            mtch.id,
-            mtch.title,
-            mtch.subtitle,
-            mtch.constraintSet,
-            mtch.status,
-            mtch.activeStatus,
-            mtch.plannedTime.execution,
-            mtch.scienceMode,
-            mtch.visualizationTime
+  extension (self: ObsEditQuery.Data.type)
```

Comment:
Maybe we can define the extension over `ObsEditQuery.Data` and make `asObsEditData` return an `ObsEditData` directly. Or do we need the `Getter`?

Reply:
Good idea